x-pack/packages/kbn-slo-schema/src/rest_specs/routes/fetch_historical_summary.ts

```diff
@@ -31,7 +31,13 @@ const fetchHistoricalSummaryParamsSchema = t.type({
           groupBy: allOrAnyStringOrArray,
           revision: t.number,
         }),
-        t.partial({ remoteName: t.string }),
+        t.partial({
+          remoteName: t.string,
+          range: t.type({
+            from: t.string,
```

Comment:
@shahzad31 I was wondering if we could use `dateRange` instead of string here, so that you avoid the type checking you do later on? Or does the datepicker expect it to be string?

Reply:
date picker expect it to be a string